Gemfile.lock

```diff
@@ -276,6 +276,7 @@ PLATFORMS
 DEPENDENCIES
   cocoapods (~> 1.14.2)
   fastlane (~> 2.205.2)
+  plist (~> 3.7, >= 3.7.1)
```

Comment:
I think I'm leaning towards copying plist sourcecode to the expo package to avoid forcing users to install the gem, but I'm open to better ideas 😅

Reply:
Isn't it already a dependency of `cocoapods`? It probably has some APIs to deal with plists 🤔

Reply:
I'm not sure, but the import didn't work for me until I installed it for my user – could use some help on how to bundle that with the script so it doesn't require a separate install.